documentation/staging/content/userguide/managing-operators/preparation.md

```diff
@@ -437,3 +354,81 @@ the operator REST API,
 setting operator pod labels,
 setting operator pod annotations,
 and Istio.
+
+### Special use cases
+
+If applicable, please review the following special use cases.
+
+#### How to download the Helm chart if Internet access is not available
+
+At a high-level, you use `helm pull` to download a released version of the Helm chart and move it to the machine with no Internet access,
+so that then you can run `helm install` to install the operator.  
+
+The steps are:
+1. On a machine with Internet access, to download the chart to the current directory, run `$ helm pull weblogic-operator --repo https://oracle.github.io/weblogic-kubernetes-operator/charts --destination .`
+2. Move the resulting `weblogic-operator-<version>.tgz` file to the machine without Internet access on which you want to install the WebLogic Kubernetes Operator.
+3. Run `$ tar zxf weblogic-operator-<version>.tgz`.
```

Comment:
Add sentence like: "This will put the helm chart files in local directory "./weblogic-operator".